src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        for (int i = 0; i < 3; i++) {
+            System.out.println(new Lottery().getRandomBall());
```

Comment:
It's better not to create several instances of the same class if you can use only one instance for your purposes